doc/glossary.rst

```diff
@@ -76,13 +76,31 @@ MNE-Python core terminology and general concepts
         down to a change of the ``first_samp`` attribute to know when cropped data
         was acquired.
 
+    forward solution
+        The forward solution or forward model (abbr. ``fwd``) is a linear
```

Comment:
```suggestion
        The forward solution (abbr. ``fwd``) is a linear
```

Reply:
agree, they're not interchangeable in my mind either